commons-numbers-complex/src/main/java/org/apache/commons/numbers/complex/Complex.java

```diff
@@ -1637,7 +1353,7 @@ public Complex tan() {
         // Define in terms of tanh
```

Comment:
This comment is not relevant any more. However it is missing from ComplexFunctions so you should move it there.